src/test/resources/soop_sst_trajectory.xml

```diff
@@ -6,68 +6,849 @@
+		<attribute name="long_name" value="'quality flags for wind direction (relative to moving platform) in the atmosphere'"/>
+		<attribute name="quality_control_set" value="3."/>
+        <attribute name="quality_control_conventions" value="'BOM (SST and Air-Sea flux) quality control procedure'"/>
+        <attribute name="flag_values" value="'B, C, D, E, F, G, H, J, K, L, M, Q, S, T, U, V, X, Z'"/>
+        <attribute name="flag_meaning" value="'Value_out_of_bounds Time_not_sequential Failed_T_Tw_Td_test Failed_true_wind_recomputation_test Platform_velocity_unrealistic Value_exceeds_threshold Discontinuity Erroneous_value Suspect_value_(visual) Value_located_over_land Instrument_malfunction Pre-flagged_as_suspect Spike_in_data_(visual) Time_duplicate Suspect_value_(statistical) Step_in_data_(statistical) Spike_in_data_(statistical) Value_passed_all_tests'"/>
+       </attributes>
+    </variable>
+        <variable>
+      <name>PL_WSPD</name>
+      <encoder>float</encoder>
+      <dimensions>
+        <dimension name="TIME"/>
+      </dimensions>
+      <attributes>
+       	<attribute name="standard_name" value="''"/>
+		<attribute name="long_name" value="'wind speed (relative to moving platform) in the atmosphere'"/>
+		<attribute name="units" value="''meter second-1''"/>
+		<attribute name="_FillValue" value="-9999.f"/>
+		<attribute name="ancillary_variables" value="'PL_WSPD_quality_control'"/>
+      </attributes>
+    </variable>
+   <variable>
+      <name>PL_WSPD_quality_control</name>
+      <encoder>char</encoder>
+      <dimensions>
+        <dimension name="TIME"/>
+      </dimensions>
+      <attributes>
+       	<attribute name="standard_name" value="''"/>
+		<attribute name="long_name" value="'quality flags for wind direction (relative to moving platform) in the atmosphere'"/>
+		<attribute name="quality_control_set" value="3."/>
+        <attribute name="quality_control_conventions" value="'BOM (SST and Air-Sea flux) quality control procedure'"/>
+        <attribute name="flag_values" value="'B, C, D, E, F, G, H, J, K, L, M, Q, S, T, U, V, X, Z'"/>
+        <attribute name="flag_meaning" value="'Value_out_of_bounds Time_not_sequential Failed_T_Tw_Td_test Failed_true_wind_recomputation_test Platform_velocity_unrealistic Value_exceeds_threshold Discontinuity Erroneous_value Suspect_value_(visual) Value_located_over_land Instrument_malfunction Pre-flagged_as_suspect Spike_in_data_(visual) Time_duplicate Suspect_value_(statistical) Step_in_data_(statistical) Spike_in_data_(statistical) Value_passed_all_tests'"/>
+       </attributes>
+    </variable>
+    <variable>
+      <name>PSAL</name>
+      <encoder>float</encoder>
+      <dimensions>
+        <dimension name="TIME"/>
+      </dimensions>
+      <attributes>
+        <attribute name="standard_name" value="'sea_water_salinity'"/>
+        <attribute name="long_name" value="'practical salinity of the water body'"/>
+        <attribute name="units" value="'PSU'"/>
```

Comment:
Salinity CF unit is 1e-3. By the way would be good to check all your units against CF.